src/sentry/utils/sdk_crashes/cocoa_sdk_crash_detector.py

```diff
@@ -1,10 +1,44 @@
 from typing import Any, Mapping, Sequence
 
+from packaging.version import InvalidVersion, Version
+
+from sentry.db.models import NodeData
 from sentry.utils.glob import glob_match
+from sentry.utils.safe import get_path
 from sentry.utils.sdk_crashes.sdk_crash_detector import SDKCrashDetector
 
 
 class CocoaSDKCrashDetector(SDKCrashDetector):
+    def should_detect_sdk_crash(self, event_data: NodeData) -> bool:
+        sdk_name = get_path(event_data, "sdk", "name")
+        if sdk_name and sdk_name != "sentry.cocoa":
+            return False
+
+        sdk_version = get_path(event_data, "sdk", "version")
+        if not sdk_version:
+            return False
+
+        try:
+            # Since changing the debug image type to macho (https://github.com/getsentry/sentry-cocoa/pull/2701)
+            # released in sentry-cocoa 8.2.0 (https://github.com/getsentry/sentry-cocoa/blob/main/CHANGELOG.md#820),
+            # the frames contain the full paths required for detecting system frames in is_system_library_frame.
+            # Therefore, we require at least sentry-cocoa 8.2.0.
+            minimum_cocoa_sdk_version = Version("8.2.0")
```

Comment:
What do you think about extracting this minimum version to a property of the class, outside the method?

Reply:
I don't mind. I like the idea.